src/LeaseCommon.h

```diff
@@ -0,0 +1,52 @@
+/* Copyright (c) 2015 Stanford University
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR(S) DISCLAIM ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL AUTHORS BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#ifndef RAMCLOUD_LEASECOMMON_H
+#define RAMCLOUD_LEASECOMMON_H
+
+namespace RAMCloud {
+
+namespace LeaseCommon {
+
+/// Defines the length of time (in nanoseconds) that a lease will be extended
+/// upon renewal.
+const uint64_t LEASE_TERM_US = 300*1e6;      // 5 min = 300,000,000 us
```

Comment:
The comment says nanoseconds, but the variable name says microseconds. Which one is it?